sentinelsat/scripts/cli.py

```diff
@@ -231,6 +244,11 @@ def cli(
         with open(os.path.join(path, "search_footprints.geojson"), "w") as outfile:
             outfile.write(gj.dumps(footprints_geojson))
 
+    if location is not None:
+        wkt, place_name = placename_to_wkt(location)
+        click.echo("The location we are querying is: '{}', '{}'".format(place_name, wkt))
+        search_kwargs["area"] = wkt
+
```

Comment:
Oh, one miiiinor issue still... this piece of code has no effect since it is run *after* the query.
Please move it before
```python
    if geometry is not None:
        search_kwargs["area"] = geojson_to_wkt(read_geojson(geometry))
```
so the `--geometry` option takes precedence if both it and `--location` are set.

Reply:
for geometry to take precedence shouldn't it be before location?

Reply:
With it before `--geometry`, if both are set then the value from `--geometry` is the final one applied to `search_kwargs["area"]`, so its the one that gets used and takes precedence.

Reply:
Ah I see ok